src/shims/unix/fd.rs

```diff
@@ -58,6 +62,19 @@ pub trait FileDescription: std::fmt::Debug + Any {
         // so we use a default impl here.
         false
     }
+
+    // Check the readiness of epoll-supported file description.
```

Comment:
Please make this a doc comment.

Also, please explain what this means. The function has no return value and no further arguments so looking just at this trait, it is entirely unclear what one is supposed to do here.